app/src/main/res/layout/loyalty_card_edit_activity.xml

```diff
@@ -239,7 +245,152 @@
+
+                    <androidx.cardview.widget.CardView
+                        android:id="@+id/reminderCardView"
+                        android:layout_width="0dp"
+                        android:layout_height="wrap_content"
+                        app:cardCornerRadius="5dp"
+                        app:cardElevation="1dp"
+                        android:padding="5dp"
+                        android:layout_weight="70"
+                        android:layout_marginBottom="2dp"
+                        android:layout_marginEnd="15dp"
+                        android:visibility="visible">
+
+                        <RelativeLayout
+                            android:layout_width="match_parent"
+                            android:layout_height="match_parent">
+
+                            <TextView
+                                android:id="@+id/daysTextView"
+                                android:layout_width="wrap_content"
+                                android:layout_height="wrap_content"
+                                android:textColor="@android:color/black"
+                                android:textSize="16sp"
+                                android:layout_marginStart="10dp"
+                                android:layout_marginTop="5dp" />
+
+                            <TextView
+                                android:id="@+id/daysBeforeTextView"
+                                android:layout_width="wrap_content"
+                                android:layout_height="wrap_content"
+                                android:textSize="16sp"
+                                android:textColor="@android:color/black"
+                                android:layout_marginTop="5dp"
+                                android:paddingStart="4dp"
+                                android:paddingEnd="1dp"
+                                android:paddingBottom="5dp"
+                                android:text="@string/days_before_expiry_date"
+                                android:layout_toEndOf="@id/daysTextView"/>
+
+                            <TextView
+                                android:id="@+id/dateReminderView"
+                                android:layout_width="wrap_content"
+                                android:layout_height="wrap_content"
+                                android:textSize="14sp"
+                                android:layout_centerVertical="true"
+                                android:layout_marginStart="10dp"
+                                android:layout_marginBottom="5dp"
+                                android:layout_below="@id/daysBeforeTextView"/>
+
+                            <TextView
+                                android:id="@+id/timeReminderTextView"
+                                android:layout_width="wrap_content"
+                                android:layout_height="wrap_content"
+                                android:textColor="@android:color/black"
```

Comment:
This should probably be something like
```
android:textColor="?android:attr/textColorSecondary"
```
so it doesn't look all wrong:
![image](https://user-images.githubusercontent.com/1885159/141515867-5e0a4fd0-ff7e-48a4-88b6-44fd5cde83b2.png)

(Or maybe you don't need to set textColor at all, needs testing)